lib/subprocess.js

```diff
@@ -146,7 +148,19 @@ class SubProcess extends EventEmitter {
         this.proc.kill('SIGINT');
 
         if (err.errno === 'ENOENT') {
-          err = new Error(`Command '${this.cmd}' not found. Is it installed?`);
+          try {
```

Comment:
could this duplicated code not be factored out into a function?

Reply:
sure, done